changelog.txt

```diff
@@ -19,12 +19,15 @@ that repo.
 - `warn-stealers`: warn when creatures that may steal your food, drinks, or items become visible
 
 ## Fixes
+- `devel/query`: updates call to deleted function that was causing an error for tile data
 - `gui/unit-info-viewer`: fix logic for displaying undead creature names
 - `gui/workflow`: restore functionality to the add/remove/order hotkeys on the workflow status screen
 - `emigration`: fix emigrant logic so unhappy dwarves leave as designed
 - `gui/gm-unit`: allow ``+`` and ``-`` to adjust skill values as intended instead of letting the filter intercept the characters
 
 ## Misc Improvements
+- `devel/query`: improved usability by informing the user when a query has been truncated
+- `devel/query`: increased default maxlength value from 257 to 2048
```

Comment:
ha, I should probably go back and modify all the example `devel/query` commands I put in the help text. I had added explicit `--maxlenth` parameters because the default was so small.

Reply:
I picked the number, mostly, out of thin air. I wasn't sure what the longest standard tables users would be looking at would be. What I did know is there are some crazy long tables that really slow down searches.
I presume I can resolve this though, or should I make a note in wherever that doc pr is?